dowhy/causal_refuters/dummy_outcome_refuter.py

```diff
@@ -163,15 +196,48 @@ def refute_estimate(self):
         dummy_estimator = copy.deepcopy(self._estimate)
         dummy_estimator.value = 0
 
-        refute.add_significance_test_results(
-            self.test_significance(dummy_estimator, sample_estimates)
-        )
+        # refute.add_significance_test_results(
+        #     self.test_significance(dummy_estimator, sample_estimates)
+        # )
 
         return refute
+
+    def data_preprocess(self):

```

Comment:
preprocess_data is better. convention is to keep function names as verbs. Also the name does not match what the method does. separate_data_by_treatment? or something like that?